apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/OpenTelemetryResourceTest.java

```diff
@@ -316,5 +322,55 @@ void sendProtobufTraces(List<Span> otelSpans, String projectName, String workspa
 
             sendBatch(payload, "application/x-protobuf", projectName, workspaceName, apiKey, expected, errorMessage);
         }
+
+        @Test
+        void testRuleMapping() {
+            var attributes = List.of(
+                    KeyValue.newBuilder().setKey("model_name").setValue(AnyValue.newBuilder().setStringValue("gpt-4o"))
+                            .build(),
+                    KeyValue.newBuilder().setKey("code.line").setValue(AnyValue.newBuilder().setIntValue(11)).build(),
+                    KeyValue.newBuilder().setKey("input")
+                            .setValue(AnyValue.newBuilder().setStringValue("{\"key\": \"value\"}")).build(),
+                    KeyValue.newBuilder().setKey("tools")
+                            .setValue(AnyValue.newBuilder().setStringValue("[\"key\", \"value\"]")).build(),
+                    KeyValue.newBuilder().setKey("all_messages")
+                            .setValue(AnyValue.newBuilder().setStringValue("[\"key\", \"value\"]")).build(),
+                    KeyValue.newBuilder().setKey("tool_responses")
+                            .setValue(AnyValue.newBuilder().setStringValue("[\"key\", \"value\"]")).build(),
+
+                    KeyValue.newBuilder().setKey("smolagents.single")
+                            .setValue(AnyValue.newBuilder().setStringValue("value")).build(),
+                    KeyValue.newBuilder().setKey("smolagents.node")
+                            .setValue(AnyValue.newBuilder().setStringValue("{\"key\": \"value\"}")).build(),
```

Comment:
Can we create a dto to which cab map to this attributes list?  that will allow us to use the podam to generate the values to be asserted